src/commentbot/parse_comment.jl

```diff
@@ -17,7 +17,8 @@ The old syntax `action(key1=val1)` is supported for backwards compatibility.
 """
 function parse_comment(text::AbstractString)
     # Handling leading ( is easy, but not capturing the closing one is a bit harder.
-    text = strip(rstrip(text, ')'))
+    lines = eachline(IOBuffer(text))
+    text = join(map(line -> rstrip(line, [' ', ')']), lines), "\n")
```

Comment:
It is definitely not needed here but if this code does get to the point where bracket matching is required you can use recursive regular expressions to handle that case: `r"^(?<paren>\((?:[^()]*|(?&paren))\))$"`

Reply:
Could be simplified to:
```julia
    text = replace(text, r"[\)\s]+$"m => "")
```